deploy/charts/harvester/dependency_charts/harvester-network-controller/templates/rbac.yaml

```diff
@@ -7,21 +7,6 @@ metadata:
   {{- include "harvester-network-controller.labels" . | nindent 4 }}
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: {{ include "harvester-network-controller.name" . }}
-rules:
-  - apiGroups: [ "network.harvesterhci.io" ]
-    resources: [ "clusternetworks", "nodenetworks" ]
-    verbs: [ "get", "watch", "list", "update", "create", "delete" ]
-  - apiGroups: [ "k8s.cni.cncf.io" ]
-    resources: [ "network-attachment-definitions" ]
-    verbs: [ "get", "watch", "list", "update" ]
-  - apiGroups: [ "" ]
-    resources: [ "nodes", "configmaps", "events" ]
-    verbs: [ "get", "watch", "list", "update", "create" ]
```

Comment:
The cluster-admin ClusterRole exists by default in your Kubernetes cluster

Reply:
will try to fix the RBAC after bump crd extendtion to v1

Reply:
What's the motivation for changing to use cluster-admin?

Reply:
@gitlawr  There is also a cost to maintaining ClusterRole, developers usually use kubeconfig (with admin privileges) when developing, when developers make changes to the code, if they forgot to update the clusterrole, then it is likely to create a bug that is not easy to find. and need to re-package the helm chart package, to The steps are also very tedious if you want to test it completely. If you use cluster-admin, then you can just modify the code.

Reply:
@gitlawr Of course, this time the change is not for convenience, but because the current dependency of wranger crd ext v1 and v1beta1 compatibility issues, if not changed to cluster-admin, controller will not work.